charts/value-dev.yml

```diff
@@ -1,8 +1,8 @@
-zookeeper:
+zenko-zookeeper:
   Image: localhost:5000/zookeeper
   ImageTag: latest
   storageClass: local-storage
-  storage: 20G
+  storage: 10G
```

Comment:
Note: before, this change was made inside the `zookeeper` Chart. Better to put it here.